player.rb

```diff
@@ -0,0 +1,63 @@
+class Player
+	def initialize(type = HUMAN)
+		@type = type
+	end
+
+	def generate_code
+		if @type == COMPUTER
+			temp_arr = ["A", "B", "C", "D", "E", "F", "G"]
+			sample_code = []
+			4.times do
+				sample_code << temp_arr.sample
```

Comment:
Loop can be avoided:
` sample_code = ["A", "B", "C", "D", "E", "F", "G"].sample(4)`
This is already used in the method generate_guess and can be reused